test_framework/Jenkinsfile

```diff
@@ -124,8 +126,12 @@ node {
                                        --env TF_VAR_tf_workspace=${TF_VAR_tf_workspace} \
                                        --env TF_VAR_tf_selinux_mode=${SELINUX_MODE} \
                                        --env TF_VAR_registration_code=${REGISTRATION_CODE} \
+                                       --env TF_VAR_gcp_project=${GCP_PROJECT} \
+                                       --env TF_VAR_gcp_auth_file=/src/longhorn-tests/gcp_creds.json \
                                        ${imageName}
                 """
+
+                sh "docker cp ${GCP_CREDS_FILE} ${JOB_BASE_NAME}-${BUILD_NUMBER}:/src/longhorn-tests/gcp_creds.json"
```

Comment:
This will only execute when `GCP_CREDS_FILE` exists.

Reply:
Yes, `GCP_CREDS_FILE` is a secret stored in Jenkins, just like the AWS credentials. https://github.com/longhorn/longhorn-tests/pull/1312/files#diff-07a02be3672fe9e5e5da2e99ceb6f30f23b939d8a849521cdb250a007276e8edR33